### ISSUE TITLE
Missing bypass and summer mode setting causes error

### ISSUE THREAD

**Issue by DTekNOdev** [bolkedebruin/erhv-lovelace#3]
I am using the Flexit Bacnet integration, which doesn't provide **summer mode**, or **bypass mode** options. The system doesn't have such a feature.

Missing out these options causes an error which is only visible in the developer console. The graphic in this case does not show at all.

![image](https://github.com/bolkedebruin/erhv-lovelace/assets/8317651/184d64b3-45e5-4f8b-a9c7-2ed0775f3851)

It would be preferable if the erhv card could detect that the attributes are not set and not display the icons for the missing attributes.

For now I am using the heater mode for summer and bypass modes.

    type: custom:erhv-card
    entity: climate.flexit_nordic
    icon: mdi:account
    supply_temperature: sensor.hvacfnct21y_a_supply_air_temperature
    intake_temperature: sensor.hvacfnct21y_a_outside_air_temperature
    return_temperature: sensor.hvacfnct21y_a_extract_air_temperature
    filter_warning: binary_sensor.hvacfnct21y_a_air_filter_polluted
    climate_entity: climate.flexit_nordic
    intake_fan_speed_rpm: sensor.hvacfnct21y_a_supply_air_fan
    intake_fan_speed_percentage: sensor.hvacfnct21y_a_supply_air_fan_control_signal 
    exhaust_fan_speed_rpm: sensor.hvacfnct21y_a_exhaust_air_fan
    exhaust_temperature: sensor.hvacfnct21y_a_exhaust_air_temperature
    exhaust_fan_speed_percentage: sensor.hvacfnct21y_a_exhaust_air_fan_control_signal
    name: Flexit recirculating fan
    filter_elapsed_time: sensor.hvacfnct21y_a_air_filter_operating_time
    preheater_state: switch.hvacfnct21y_a_electric_heater
    show_empty: true
    summer_mode: switch.hvacfnct21y_a_electric_heater
    bypass_state: switch.hvacfnct21y_a_electric_heater

**Comment by bolkedebruin** [comment]
Check if it works now for you with latest master (no release yet)

**Comment by DTekNOdev** [comment]
Thank-you! I wasn't very sure how to test it locally directly from your main branch so I forked your repo, and built a (temporary) release of my own, then tested from my repo. Now if either of summer mode or bypass state are missing, there is no error and the visualization is shown. I assume the same will also be true for filter warning and preheater state, at least your code looks sound on that score.


On another note I noticed that "Filter elapsed time" shows up twice in the visual editor.

![image](https://github.com/bolkedebruin/erhv-lovelace/assets/8317651/4c66f0e6-ee4a-4ed3-9db0-38b3f2c76679)
